src/label/creator.py

```diff
@@ -119,12 +127,18 @@ def __init__(self, image, label_width=CANVAS_WIDTH):
 
 class Label(object):
 
-    def __init__(self, label_objects):
+    def __init__(self, label_objects, label_height_mm=None):
 
         self.label_width = CANVAS_WIDTH
         self.label_objects = label_objects
 
-        self.label_height = self.get_canvas_height() + ((len(self.label_objects) + 1) * IMG_MARGIN)
+        if label_height_mm is None:
+            self.label_margin = IMG_MARGIN
+            self.label_height = self.get_canvas_height() + ((len(self.label_objects) + 1) * self.label_margin)
```

Comment:
It's a bit hard to follow what units each variable is expressed in. For example, are these in pixels?

Reply:
I do agree, but that refactoring should be another issue.